Python/Tests/DebuggerTests/DebuggerTests.cs

```diff
@@ -1968,6 +1970,8 @@ private enum ExceptionMode {
                             attached.Set();
                             breakpointHit.Set();
                         }
+
+                        continueEvent.WaitOne(1000);

```

Comment:
The tricky part is asserting on the right thread (this callback gets called on the debugger background thread, which the test runner knows nothing about - if assert fails on it, it just kills the test without actually logging the assert properly).

Given that the test will basically just fail anyway if we time out here, it feels that adding a bunch more code here to propagate the assert is not worth it.